addons/io_scene_gltf2/io/exp/gltf2_io_user_extensions.py

```diff
@@ -12,11 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-def export_user_extensions(hook_name, export_settings, gltf2_object, *args):
-    if gltf2_object.extensions is None:
-        gltf2_object.extensions = {}
+def export_user_extensions(hook_name, export_settings, *args):
+    gltf2_object = args[0]
+    try:
+        object_extensions = getattr(gltf2_object, "extensions")
+    except AttributeError:
+        pass
+    else:
+        if object_extensions is None:
+            setattr(gltf2_object, "extensions", {})
```

Comment:
Easier to do

````py
if args and hasattr(args[0], 'extensions'):
    if args[0].extensions is None:
        args[0].extensions = {}
````

Reply:
Then you need to do it in all the extensions... By default, the extensions is None, and you dont know which extension is going to be called first

Reply:
Ive modify it as you said